test/zdtm/static/sigaltstack.c

```diff
@@ -14,6 +13,18 @@
 const char *test_doc	= "Check for alternate signal stack";
 const char *test_author	= "Cyrill Gorcunov <gorcunov@openvz.org>";
 
+/*
+ * The latest glibc has redefined SIGSTKSZ as 'sysconf (_SC_SIGSTKSZ)' and
+ * this breaks the static char[] definition below. According to the kernel
+ * sources all for CRIU relevant architectures use 8192 except AARCH64.
+ */
+#undef SIGSTKSZ
+#if __aarch64__ == 1
+#define SIGSTKSZ 16384
```

Comment:
We don't need to use the SIGSTKSZ constant, we can define TESTSIGSTKSZ to 16384 for all architectures.

Reply:
changed